src/libraries/System.Text.RegularExpressions/src/System/Text/RegularExpressions/RegexCharClass.cs

```diff
@@ -300,7 +300,8 @@ internal sealed class RegexCharClass
         private static readonly LowerCaseMapping[] s_lcTable = new LowerCaseMapping[]
         {
             new LowerCaseMapping('\u0041', '\u005A', LowercaseAdd, 32),
-            new LowerCaseMapping('\u00C0', '\u00DE', LowercaseAdd, 32),
+            new LowerCaseMapping('\u00C0', '\u00D6', LowercaseAdd, 32),
```

Comment:
Not you, but this file in general could use some better naming/commenting. Eg., a one line comment here explaining what each of the 4 entries are would be helpful. I look in the struct, and they're Chmin, Chmax, LcOp, Data -- which are all pretty bad names that could be improved.

Elsewhere the naming could be improved too. Eg., `LowercaseBad ` we would normally name something like `LowerCaseBitwiseAnThenAddOne` or something, and we'd probably use an enum not consts.

Not suggesting you need to fix in this PR, in fact I think it's probably better one of us do it separately later.

Reply:
There's some spiel above this spot (L267-L293), but I agree with the comment. It wasn't obvious at all what `LowercaseSet/Bor/Bad` did from the name :)